test/testdrive/sinks.td

```diff
@@ -116,3 +116,29 @@ $ kafka-verify format=avro sink=materialize.public.snk8
 {"before": null, "after": {"a": "jack", "b": "jill", "mz_line_no": 2}}
 {"before": null, "after": {"a": "goofus", "b": "gallant", "mz_line_no": 3}}
 {"before": null, "after": {"a": "extra", "b": "row", "mz_line_no": 4}}
+
+# Test that we are correctly handling WITH/WITHOUT SNAPSHOT.
+> CREATE MATERIALIZED VIEW foo AS VALUES (1), (2), (3);
+
+# Make sure data is read into view before we TAIL it.
```

Comment:
I'm surprised that this bit was necessary. `avro-sinks.td` creates a static view and immediately `SINK`s it and thats been fine afaik. if thats wrong then I can amend those tests

Reply:
it is working without the `SELECT`!